book/website/afterword/glossary.md

```diff
@@ -486,7 +486,7 @@ repo2docker
   A tool to build Docker images from code repositories.
 
 Repository
-  *Same as Data or Code Reprository*. A long-lived place on the internet where resources (be they data, software, publications or anything else) can be stored and accessed. This keyword is often shortened to ‘repo’.
+  A place on the internet where resources (data, software, publications or anything else) can be stored and accessed. This keyword is often shortened to ‘repo’. See Data Repository if this place is long-lived.
```

Comment:
Oh! Now I get why you use this wording! 😅

Reply:
I'm happy to change, but I'm not sure if other people had strong feelings about the wording! :P

Reply:
It might be good to have a general acknowledgement of how it's broadly used, i.e. "While a repository is broadly defined as a central location where resources are stored can be stored and accessed, in computing this is applied to data, software, publications or anything else. This keyword is often shorted to 'repo'.